### ISSUE TITLE
Place Tabulator Size in Status Bar, Make it Changable Per File, Implement Elastic Tabstops

### ISSUE THREAD

**Issue by tobsimon** [notepad-plus-plus/notepad-plus-plus#1712]
I frequently change the tabulator size, when working with database like files (TSV) which have variable width columns. I also frequently change whether tabulators should be converted to spaces. Different projects and different laguages have different coding guidlines in place for that. It is annoying to click 4 layers deep to do that. I'd like to do it on the status bar.
## 

It is currently possible to change the language and the line ending style through context menues on the appropriate sections of the status bar. I think there should be a new section there, which displays the current tabulator length and if they are replaced by spaces. E.g. "tab : 4 spaces" would mean length 4 and replaced by spaces. The context menu of this section allows to quickly change these values. The conversion functions should also be there.

The tabulator width should be set on a per open file basis. I'm aware, that text files do not support saving the tabulator width, other than in special non standard constructs inserted into the file. Maybe Notepad++ should keep it's own history of preferred tabulator lengths of recently closed files to restore these options, but I think this may constitude as unexpected behaviour.

For the next section I initially wanted to suggest setting tabulator lengths on a per column basis, but then I found elastic tabstops, and I think they are way awesome-er and elegant than my idea:
http://nickgravgaard.com/elastic-tabstops/
In a nutshell: Tabulators automatically align to the width of their cell.
Please implement elastic tabstops. There already is a implementation for Scintilla. I can't wait to use them in my favorite text editor.

**Comment by MetaChuh** [comment]
@tobsimon 

please use the newest elastic tabstops plugin for 32 or 64 bit notepad++ version 1.3 from here: https://github.com/dail8859/ElasticTabstops/releases

as this plugin apparently does exactly what you need, there will be no need to implement that code to the notepad++ core.

thank you for your support and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)
